### DISCUSSION TITLE
-vvv dont work, i cant see my console.log

### DISCUSSION THREAD

**Discussion by kuto3** [Cyfrin/security-and-auditing-full-course-s23#203] [Q&A] [answered]
Hi, i have a problem i have done a proof of code for the reentrancy in the puppyraffle and when i write in the terminal 
~/RaffleCourse/4-puppy-raffle-audit# forge test --match-test testReentrancy -vvv
thats just write this : 
```
[⠊] Compiling...
[⠒] Compiling 36 files with 0.7.6
[⠢] Solc 0.7.6 finished in 968.61ms
Compiler run successful!
``` 
so i have no verbose and i dont see my console.log.
if anybody know why thats done that and can help me its will be great.

**Comment by kuto3** [comment, marked as answer]
ok i found a solution, my contract was "abstract" and when i remove this, its work.

**Reply by EngrPips** [reply]
Awesome, I am glad you unblock yourself. Keep crushing it.
